docs/source/data-publishing/ogcapi-processes.rst

```diff
@@ -9,17 +9,24 @@ fashion (inputs, outputs).
 pygeoapi implements OGC API - Processes functionality by providing a plugin architecture, thereby
 allowing developers to implement custom processing workflows in Python.
 
-A `sample`_ ``hello-world`` process is provided with the pygeoapi default configuration.
+The pygeoapi offers two processes: a default ``hello-world`` process which allows you to quickly explore the capabilities of processes, and an optional ``shapely-functions`` process with more advanced features that leverages Shapely_ to expose various geometric processing functionality.
 
 Configuration
 -------------
 
 .. code-block:: yaml
 
    processes:
+    
+    # enabled by default
        hello-world:
            processor:
                name: HelloWorld
+        
+        #Add this block to the configuration file to enable shapely processes.
+        #shapely-functions:
+           #processor:
+               #name: ShapelyFunctions
```

Comment:
Could you move this to the start of the Shapely section, with the process uncommented?
```md
Shapely Functions (Optional)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^

.. code-block:: yaml

        shapely-functions:
           processor:
               name: ShapelyFunctions
```

Reply:
Alright. Will do.